components/calendar/src/types.rs

```diff
@@ -49,6 +46,10 @@ pub struct FormattableYear {
     /// The year number in the current era (usually 1-based).
     pub number: i32,
 
+    /// The year in the current cycle for cyclic calendars;
+    /// can be ignored and set to any value for non-cyclic calendars.
+    pub cyclic: i32,
```

Comment:
issue: this should probably be an `Option`, allowing formatter implementations to choose how to handle unknown cyclics

Reply:
Ok, I made this an Option<i32> with all currently existing calendars having this field set to None